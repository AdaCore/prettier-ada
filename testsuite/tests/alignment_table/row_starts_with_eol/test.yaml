title: Table_Alignment row starts with EOL
driver: document_formatter
description: |
  This test confirms that Table_Alignment does not add indentation to a row if it starts with EOL.
