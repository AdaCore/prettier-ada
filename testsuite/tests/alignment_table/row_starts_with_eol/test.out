Foooooooooooooo
  AAAAA BBBBB

  CCC   DDDD
